.jsdoc.js

```diff
@@ -0,0 +1,49 @@
+/*!
```

Comment:
We had updated all the other files to use the `//` style comments for the license header.  Could I trouble you to update the template to be consistent?

Reply:
https://github.com/googleapis/synthtool/pull/351